ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-main.yaml

```diff
@@ -3,10 +3,6 @@ base_images:
     name: stream8
     namespace: coreos
     tag: "8"
-  golangci-lint:
-    name: golangci-lint
-    namespace: ci
-    tag: v1.37.1
   openshift_release_golang-1.16:
```

Comment:
This can be bumped up to 1.17

Reply:
You're probably right, but I think we should rather not change too much behavior in this PR :)

Reply:
Well, if future it will be a taks for the ci-bot :)

Reply:
Does the ci-bot update that? That would be awesome 😀

Reply:
Yeah, there's a registry replacer for that, ie https://github.com/openshift/release/commit/48ba4f8d9745cb273048d6c89245e70bd29e1ef3